README.md

```diff
@@ -1,3 +1,56 @@
 # perspectiveapi-js-client
 
-A simple example JS/TS client library for the [perspective api](https://perspectiveapi.com/)
+A simple example JS/TS client library for the [Perspective API](https://www.perspectiveapi.com/).
+This client accepts text and returns some of the text's attributes.
+
+## Usage
+
+First, you'll need an API key from ConversationAI's [Perspective API](https://www.perspectiveapi.com/).
+
+```
+require { Client } from "perspectiveapi-js-client";
+const client = new Client(YOUR_AP_KEY);
+const results = client.getScores(text, options);
+
+```
+
+## Methods
+
+This client exposes one method, getScores.
```

Comment:
Looks like we won't be able to include that on this initial release due to time constraints.